core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -608,15 +608,19 @@ package object config {
     ConfigBuilder("spark.storage.decommission.fallbackStorage.path")
       .doc("The location for fallback storage during block manager decommissioning. " +
         "For example, `s3a://spark-storage/`. In case of empty, fallback storage is disabled. " +
-        "The storage should be managed by TTL because Spark will not clean it up.")
+        "The storage will not be cleaned up by Spark unless " +
+        "spark.storage.decommission.fallbackStorage.cleanUp is true. " +
+        "Use an external clean up mechanism when false, for instance a TTL.")
       .version("3.1.0")
       .stringConf
       .checkValue(_.endsWith(java.io.File.separator), "Path should end with separator.")
       .createOptional
 
   private[spark] val STORAGE_DECOMMISSION_FALLBACK_STORAGE_CLEANUP =
```

Comment:
This config is already present, how did we clean up shuffle from the fallback storage before?

Reply:
Currently, shuffle data are only removed from fallback storage on application exit (Spark context shutdown).